_pages/plugins/modularimageanalysis.md

```diff
@@ -0,0 +1,217 @@
+---
+title: ModularImageAnalysis (MIA)
+logo: /media/icons/mia.png
+categories: [Analysis, Automation, Scripting]
+doi: 10.5281/zenodo.1201320
+artifact: io.github.mianalysis:mia
+---
+
+{% include img align="center" name="fig:ModularImageAnalysis screenshot." src="/media/plugins/mia/main-screenshot" %}
```

Comment:
If we move your page to `_pages/plugins/modularimageanalysis/index.md`, and move the images to `/media/plugins/modularimageanalysis/`, then you can reference them simply like `src="main-screenshot"` instead of needing to put the full path. But it's not a big deal. It's just tidier for the page directory and media directory to align.